src/Rector/AbstractRector.php

```diff
@@ -190,7 +190,7 @@ final public function enterNode(Node $node): int|Node|null
      * Replacing nodes in leaveNode() method avoids infinite recursion
      * see"infinite recursion" in https://github.com/nikic/PHP-Parser/blob/master/doc/component/Walking_the_AST.markdown
      */
-    public function leaveNode(Node $node): array|int|Node|null
+    final public function leaveNode(Node $node): array|int|Node|null
```

Comment:
It seems bc break

Reply:
On purpose, this should have never been allowed.